### PR TITLE
do not blow up on _method / _utf8 etc parameters when using strict pa…

### PR DESCRIPTION
**Pull request by grosser**
…rameter verification

@jamesarosen

### CODE REVIEW

File: gemfiles/rails3_2.gemfile [hunk 1]
```diff
@@ -4,3 +4,4 @@ gemspec :path=>"../"
 
 gem "rails", "~> 3.2.18"
 gem 'minitest', "< 5"
+gem "test-unit-minitest"
```
Comment (grosser) [inline gemfiles/rails3_2.gemfile]: needed for ruby 2.2 vs rails 3.2

Comment (jamesarosen) [inline gemfiles/rails3_2.gemfile]: How odd!

File: lib/arturo/feature_params_support.rb [hunk 2]
```diff
@@ -18,14 +18,15 @@ module WithStrongParams
       PERMITTED_ATTRIBUTES = [ :symbol, :deployment_percentage ]
 
       def feature_params
-        params.permit(:feature => PERMITTED_ATTRIBUTES)[:feature]
+        if feature = params[:feature]
+          feature.permit(PERMITTED_ATTRIBUTES)
+        end
```
Comment (jamesarosen) [inline lib/arturo/feature_params_support.rb]: +1

**Comment by jamesarosen** [comment]
+1. Merge before or after #93?

**Comment by grosser** [comment]
merge while it's green :D

**Comment by grosser** [comment]
ah good the other turned green too ... either way will be a merge conflict ...do whatever

**Comment by jamesarosen** [comment]
Ready for rebase :)

**Comment by grosser** [comment]
done!

**Comment by jamesarosen** [comment]
Is it time to cut a new minor version now?

**Comment by grosser** [comment]
Yes please!

**Pull request merged**